backend/packages/graphql/schema/pagination/arangodb.js

```diff
@@ -0,0 +1,292 @@
+// @flow
```

Comment:
This code needs to be examined more closely to be sure we are not vulnerable to SQL-injection like attacks. I believe the ```bindVars``` has sanitation logic, but we are also doing some string concatenation.